tests/test_dwh.py

```diff
@@ -1,54 +1,24 @@
-import pytest
-import random
-import string
-import pyodbc
 import logging
-import pandas as pd
-
 from unittest.mock import Mock
-from pyprediktorutilities.dwh import Dwh
-from pandas.testing import assert_frame_equal
-
-"""
-Helpers
-"""
-
-
-class mock_pyodbc_connection:
-    def __init__(self, connection_string):
-        pass
-
-    def cursor(self):
-        return
-
-
-def mock_pyodbc_connection_throws_error_not_tolerant_to_attempts(connection_string):
-    raise pyodbc.DataError("Error code", "Error message")
-
-
-def mock_pyodbc_connection_throws_error_tolerant_to_attempts(connection_string):
-    raise pyodbc.DatabaseError("Error code", "Error message")
-
-
-def grs():
-    """Generate a random string."""
-    return "".join(random.choices(string.ascii_uppercase + string.digits, k=10))
 
```

Comment:
Inside tests folder create a new folder `dwh` and inside put test_dwh.py and test_dwh_singleton.py. The folder structure of the tests folder must reflect the folder structure of what we test.